projects/plugins/jetpack/modules/seo-tools.php

```diff
@@ -38,4 +36,8 @@
 	}
 }
 
-new Jetpack_SEO;
+if ( ! apply_filters( 'jetpack_disable_seo_tools', false ) ) {
+	require_once __DIR__ . '/seo-tools/jetpack-seo.php';
+	require_once ABSPATH . 'wp-admin/includes/plugin.php';
```

Comment:
> I think you'll need to keep the require at the top of this file, as we need it above when we check for active plugins.

Do you mean just the require to `plugin.php` or both?

Also, could you help me understand this a bit better. I see the call to `Jetpack::is_plugin_active`, shouldn't this just work since the Jetpack plugin is already running at this point?

Reply:
Just `plugin.php`.

> could you help me understand this a bit better. I see the call to Jetpack::is_plugin_active, shouldn't this just work since the Jetpack plugin is already running at this point?

I think it was added in 153691a3d8cf18e7b2c9f9160f696ad972003207 as a safeguard before calling core's `is_plugin_active()`, but you're right, it may indeed just work now that we use `Jetpack::is_plugin_active()`. You could remove it altogether now!

```suggestion
```